### ISSUE TITLE
text within <textarea>...</textarea> should preserve line breaks, please

### ISSUE THREAD

**Issue by nbogol** [gajus/dindent#11]

**Comment by abestic9** [comment]
also, <pre>

**Comment by horstoeko** [comment]
Same Issue here...

**Comment by gajus** [comment]
Can be easily done by identifying the `<pre>` and `<textarea>` tags and not indenting their contents.

The problem is that the text flow behaviour depends on the CSS property `line-break`. This fix will address `<textarea>` input value problem, but will not do anything about other elements that have `line-break: pre` set.

**Comment by tobia** [comment]
Yes, the CSS property `line-break` makes the problem practically undecidable.

As a workaround, we could use a configuration option to set the tag names that must be preserved, with a default of `['script', 'textarea', 'pre']`

**Comment by aaronpk** [comment]
+1 to a configuration option for skipping specific tags. I ran into this problem with `<pre>` tags today.
